scripts/sesar_indexing_test_records.json

```diff
@@ -0,0 +1 @@
212, 0.0001111503588617, 5.761328338849125e-06]}}, "826371": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998681545257568, 0.0001056277615134, 5.236950073594926e-06]}}, "134353": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998395442962646, 0.0001310749212279, 5.632558895740658e-06]}}, "749185": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998251795768738, 0.0001439881307305, 5.999808308843058e-06]}}, "142804": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998329877853394, 0.0001372199039906, 5.951682396698743e-06]}}, "1291443": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998268485069276, 0.0001376177242491, 8.006630196177866e-06]}}, "196333": {"material": {"values": ["Sediment", "Rock", "Other"], "confidence": [0.9998408555984496, 0.0001296736299991, 5.566838808590546e-06]}}, "143660": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998270869255066, 0.0001421218184987, 5.860509645572165e-06]}}, "694329": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998328685760498, 0.0001371170656057, 5.838235665578395e-06]}}, "727877": {"material": {"values": ["Sediment", "Rock", "Other"], "confidence": [0.999855637550354, 0.0001168267481261, 5.331257852958515e-06]}}, "668280": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.999837040901184, 0.0001332468382315, 5.6145249800465535e-06]}}, "1506087": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998325109481812, 0.0001371937978547, 5.96473137193243e-06]}}, "1647328": {"material": {"values": ["Sediment", "Rock", "Other"], "confidence": [0.8730365037918091, 0.1065632104873657, 0.014138288795948]}}, "2764874": {"material": {"values": ["Sediment", "Rock", "Other"], "confidence": [0.9998544454574584, 0.0001176349178422, 5.618644991045585e-06]}}, "887118": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9985350370407104, 0.0013075263705104, 7.010786066530272e-05]}}, "884704": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998680353164672, 0.0001056243199855, 5.146100647834828e-06]}}, "530434": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998403787612916, 0.0001304679462919, 5.714818144042511e-06]}}, "106950": {"material": {"values": ["Sediment", "Rock", "Other"], "confidence": [0.9998619556427002, 0.0001108620126615, 5.498380232893396e-06]}}, "1045111": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.999835968017578, 0.0001339845912298, 5.788242560811341e-06]}}, "1289169": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9997649788856506, 0.0001859765470726, 1.1132806321256794e-05]}}, "461563": {"material": {"values": ["Sediment", "Rock", "Biology"], "confidence": [0.9998677968978882, 0.0001060401264112, 5.197081463848008e-06]}}}
```

Comment:
Construct json for test records of SESAR. When rule-based classification is done, there is only one label and confidence value would be an array with one value -1. If machine-based classification is conducted, top 3 predicted results are contained in `values` and each confidence score of it is contained in `confidence`.

Reply:
Could we format this so it isn't a single line, please?  Here it is:

```
{
	"ODP02WZQI": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02U76S": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02TOEW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP031XC1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02W8WY": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP023P52": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01CRRU": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02J3QK": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02234Z": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WVXC": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01T3DH": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02XHJW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WO5M": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP025SEV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP026DBU": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02RQES": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01YBND": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02HW59": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01A4WR": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02KRP4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01CT6A": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02V4QX": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01WLJN": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02U2SA": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02IY8S": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02D9CZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02STS0": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01X18P": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01UI6L": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP031LR7": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02UU02": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KXUV": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP025Y18": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01LWOW": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02X3C2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02AABS": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02OM0H": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01X07F": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02XPZC": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02P0WX": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02NAN3": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022MEY": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02TZYM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP020W8Z": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01SL2N": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP021GI4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP03151T": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022ER3": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01D873": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DOET": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP030Y4W": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0227KK": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP030GB6": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01GICZ": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP029NGM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02A4F9": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02DVH2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DAJQ": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0268HO": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01VCEQ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0326P6": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WSGZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01FZV5": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02RGPO": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02GBS4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01ASO6": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01PY24": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02HV6W": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02Z65I": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01A3E4": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01E590": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02458I": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02V8YS": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP030LPI": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WIET": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01D3U9": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP021BYL": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02CW17": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02POLA": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP027XG0": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02FJIW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP023XRZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KJN3": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP029V17": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01P8US": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022QJB": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02SH6V": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KPAD": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01E77L": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP030CNV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02H2V1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01WQVZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01MW6K": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02KCVE": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01CN4R": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP029O1A": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01C4WL": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Y9QG": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01B71X": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KMML": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XV1P": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WPZ7": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XXGE": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02X15I": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0243O4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP019HK3": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02XFC4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01TXEA": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02LNC7": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP025KSN": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02OAMD": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02ECOV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0255NK": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP028JDG": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KMLX": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP030A7A": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01EQH2": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02RXEZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP023MFF": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01N4NU": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Y1JW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02HOST": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02UB9U": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01V2C4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02Z0GQ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DL4Q": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01UHEJ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02LUIR": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01O4PY": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02XFEA": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02PDM3": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01X1PM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Z1S1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Z714": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP03175Y": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02ARPS": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP028WNE": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02PZ9J": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02LYP6": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02OWYV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01GW8S": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01MB8V": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01D1LI": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0328RT": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DHQB": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02KPW9": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP028GGX": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02IOQW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02IPUM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022LJI": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP026N9G": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02V67R": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP032MTL": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02HL6S": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0249G2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02MTU2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02NFUP": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02FJ3O": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01ZWJF": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01IUSE": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DLVV": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01V3F7": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01BJQZ": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0219ID": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP021BEW": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02SUMH": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01OF4V": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02S038": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01RLSH": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP024XYZ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02IFV6": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02LWGR": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0316G2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01BXPO": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XDTU": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01V5E4": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01GEMG": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01BVAE": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP023RK8": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02BXDL": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02XR9U": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XLUC": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP019D71": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02KSKJ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01TT8E": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01A921": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02V5AS": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01WUAM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XMAH": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02CYL1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"IESFS00J2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XOKK": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01XRYF": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0280W9": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02DA9M": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP0283AD": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02VFHL": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02957J": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02UQZV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02SUGX": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02YCW0": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01VUHM": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01U304": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01X9GO": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP027S8I": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01F4H6": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02WFKU": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP020Q0E": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02IK1B": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP028QC2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02FVG8": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"IESFS051V": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02C8ED": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01B41L": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01ATQS": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02NMF2": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP032L0T": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01OEF4": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02PF41": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022W5Q": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02OOOV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022Y6Z": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP025HLR": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"IESFS03OA": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022A3C": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01KT5E": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Q9AH": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02VPB1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01V4BG": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP021T4Y": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01I2VC": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02K54B": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02JSUV": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP028PH1": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01Y9HJ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02DGHK": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01WBSX": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02JZBO": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02H6LD": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02Z303": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02DUDI": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP021W57": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP010F1F": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02Q2G7": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01DWT9": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01D1KL": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02KYPJ": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02PZ9V": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02Q6P5": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02ZPMS": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP01FIS7": {
		"material": {
			"values": [
				"Mixed soil, sediment, rock"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP022S0P": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"ODP02CV44": {
		"material": {
			"values": [
				"Natural Solid Material"
			],
			"confidence": [
				-1
			]
		}
	},
	"197554": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998408555984496,
				0.0001296733826166,
				5.566827894654125e-06
			]
		}
	},
	"2222707": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998406171798706,
				0.0001301405136473,
				5.474320460052695e-06
			]
		}
	},
	"789185": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998288154602052,
				0.0001324926706729,
				8.83905067894375e-06
			]
		}
	},
	"1882085": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9997877478599548,
				0.000169340026332,
				1.1254714081587736e-05
			]
		}
	},
	"3146848": {
		"material": {
			"values": [
				"Rock",
				"Sediment",
				"Other"
			],
			"confidence": [
				0.9994152784347534,
				0.0001871549466159,
				0.0001400570035912
			]
		}
	},
	"2925728": {
		"material": {
			"values": [
				"Rock",
				"Sediment",
				"Other"
			],
			"confidence": [
				0.9810721278190612,
				0.0146950939670205,
				0.0018780787941068
			]
		}
	},
	"46892": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998617172241212,
				0.0001113637335947,
				5.363729997043265e-06
			]
		}
	},
	"3083915": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998389482498168,
				0.0001316002890234,
				5.803774911328219e-06
			]
		}
	},
	"2125771": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998409748077391,
				0.0001297115959459,
				5.457750830828445e-06
			]
		}
	},
	"164834": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.999842882156372,
				0.0001281930890399,
				5.542863164009759e-06
			]
		}
	},
	"165154": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.999842882156372,
				0.0001281930890399,
				5.542863164009759e-06
			]
		}
	},
	"941784": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998668432235718,
				0.0001067305347532,
				5.133243121235864e-06
			]
		}
	},
	"1053246": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998668432235718,
				0.0001065242104232,
				5.471572876558639e-06
			]
		}
	},
	"3241254": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998995065689088,
				7.682765135541558e-05,
				4.201683168503223e-06
			]
		}
	},
	"410588": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998389482498168,
				0.0001316002890234,
				5.803774911328219e-06
			]
		}
	},
	"142007": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998644590377808,
				0.0001090505174943,
				5.3899989325145725e-06
			]
		}
	},
	"1019213": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998675584793092,
				0.0001060142094502,
				5.119088655192172e-06
			]
		}
	},
	"2765283": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998633861541748,
				0.0001095027982955,
				5.582509857049445e-06
			]
		}
	},
	"2031351": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9993639588356018,
				0.0005194146069698,
				3.147818279103376e-05
			]
		}
	},
	"1862367": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.999864935874939,
				0.0001081482841982,
				5.719537966797361e-06
			]
		}
	},
	"142281": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998644590377808,
				0.0001090505174943,
				5.3899989325145725e-06
			]
		}
	},
	"188314": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998416900634766,
				0.0001289637293666,
				5.571670953941066e-06
			]
		}
	},
	"1291410": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9994571805000304,
				0.0004378509765956,
				3.0721170332981274e-05
			]
		}
	},
	"3056437": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.999868392944336,
				0.0001051866856869,
				5.298930773278698e-06
			]
		}
	},
	"3346231": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998328685760498,
				0.0001370917016174,
				5.795393917651381e-06
			]
		}
	},
	"1311819": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998459815979004,
				0.0001250854256795,
				5.498019618244143e-06
			]
		}
	},
	"2600648": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998617172241212,
				0.0001111503588617,
				5.761328338849125e-06
			]
		}
	},
	"826371": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998681545257568,
				0.0001056277615134,
				5.236950073594926e-06
			]
		}
	},
	"134353": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998395442962646,
				0.0001310749212279,
				5.632558895740658e-06
			]
		}
	},
	"749185": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998251795768738,
				0.0001439881307305,
				5.999808308843058e-06
			]
		}
	},
	"142804": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998329877853394,
				0.0001372199039906,
				5.951682396698743e-06
			]
		}
	},
	"1291443": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998268485069276,
				0.0001376177242491,
				8.006630196177866e-06
			]
		}
	},
	"196333": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998408555984496,
				0.0001296736299991,
				5.566838808590546e-06
			]
		}
	},
	"143660": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998270869255066,
				0.0001421218184987,
				5.860509645572165e-06
			]
		}
	},
	"694329": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998328685760498,
				0.0001371170656057,
				5.838235665578395e-06
			]
		}
	},
	"727877": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.999855637550354,
				0.0001168267481261,
				5.331257852958515e-06
			]
		}
	},
	"668280": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.999837040901184,
				0.0001332468382315,
				5.6145249800465535e-06
			]
		}
	},
	"1506087": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998325109481812,
				0.0001371937978547,
				5.96473137193243e-06
			]
		}
	},
	"1647328": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.8730365037918091,
				0.1065632104873657,
				0.014138288795948
			]
		}
	},
	"2764874": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998544454574584,
				0.0001176349178422,
				5.618644991045585e-06
			]
		}
	},
	"887118": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9985350370407104,
				0.0013075263705104,
				7.010786066530272e-05
			]
		}
	},
	"884704": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998680353164672,
				0.0001056243199855,
				5.146100647834828e-06
			]
		}
	},
	"530434": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998403787612916,
				0.0001304679462919,
				5.714818144042511e-06
			]
		}
	},
	"106950": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Other"
			],
			"confidence": [
				0.9998619556427002,
				0.0001108620126615,
				5.498380232893396e-06
			]
		}
	},
	"1045111": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.999835968017578,
				0.0001339845912298,
				5.788242560811341e-06
			]
		}
	},
	"1289169": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9997649788856506,
				0.0001859765470726,
				1.1132806321256794e-05
			]
		}
	},
	"461563": {
		"material": {
			"values": [
				"Sediment",
				"Rock",
				"Biology"
			],
			"confidence": [
				0.9998677968978882,
				0.0001060401264112,
				5.197081463848008e-06
			]
		}
	}
}
```